docs/Selectors/Value selector.md

```diff
@@ -0,0 +1,6 @@
+# Value selector
+Value selector is input constant wich will be added to parent selector, like column with name selector "id" and contain "input constant"
```

Comment:
This description is for Constant Value. We should add it in separate file and change this description for Selector Value description.